upload/admin/model/user/user.php

```diff
@@ -175,7 +217,47 @@ public function getUsers(array $data = []): array {
 	 * @return int
 	 */
 	public function getTotalUsers(): int {
-		$query = $this->db->query("SELECT COUNT(*) AS `total` FROM `" . DB_PREFIX . "user`");
+		$sql = "SELECT COUNT(*) AS `total` FROM `" . DB_PREFIX . "user` `u` ";
+
+		$implode = [];
+
+		if (!empty($data['filter_username'])) {
+			$implode[] = "LCASE(`u`.`username`) LIKE '" . $this->db->escape(oc_strtolower($data['filter_username']) . '%') . "'";
```

Comment:
This function does not have a $data variable so you need to add it, maybe via a function argument like it is for `getUsers()`.

```php
public function getTotalUsers(array $data = []): int {
```